code/__HELPERS/text.dm

```diff
@@ -1,4 +1,4 @@
-/*
+aa/*
```

Comment:
Loose letters.

Reply:
dont know how they got in there, but uploaded a cleaned file, thank you